lib/datapacksexpand.js

```diff
@@ -425,6 +425,15 @@ DataPacksExpand.prototype.getNestedDataPackReferences = function(currentData, pa
                 }
                 
             } else {
+                if(!(jobInfo.maxDepth === 0) || jobInfo.includeSalesforceMetadata){
```

Comment:
This should actually just be `if (jobInfo.maxDepth != 0)` - Then in the apexclass.js it should check the 

`jobInfo.includeSalesforceMetadata` to make sure of whether or not to return a valid value. If jobInfo.includeSalesforceMetadata is false it could just return null.

Reply:
Code has been updated but Why are we waiting for apexclass.js to return null, if it can be skipped in datapacksexpand.js itself?

Reply:
Because the check for `jobInfo.includeSalesforceMetadata` only applies to Salesforce Metadata not Vlocity DataPacks. So if a Vlocity DataPack onAutoExport handler were added then this check would no longer be valid there.